libs/application/templates/data-protection-complaint/src/forms/ComplaintForm.ts

```diff
@@ -313,7 +313,7 @@ export const ComplaintForm: Form = buildForm({
                   title: info.labels.nationalId,
                   format: '######-####',
                   width: 'half',
-                  backgroundColor: 'blue',
+                  backgroundColor: 'white',
```

Comment:
Shouldn't address, postal code and city also have white background?

Reply:
The request was that only non editable fields should have white backgrounds